src/api/custom-schema/type-defs.ts

```diff
@@ -198,6 +198,8 @@ const typeDefs = `
     hasMinimumUkGoodsOrServices: Boolean!
     totalContractValueId: Int!
     validExporterLocation: Boolean!
+    isPartyToConsortium: Boolean!
+    isMemberOfAGroup: Boolean!
```

Comment:
Can we update `ApplicationEligibility` in the API's `types/application-types` as well?